### PR TITLE
Papa 9a and Irminger 9 glider cals

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: calibration/CTDGVM/CGINS-CTDGVM-09236__20140923.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9236,CC_calibration_date,1411430400000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/CTDGVM/CGINS-CTDGVM-09236__20140923.csv:2]: looks good

File: calibration/CTDGVM/CGINS-CTDGVM-09236__20200213.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9236,CC_calibration_date,1581552000000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/CTDGVM/CGINS-CTDGVM-09236__20200213.csv:2]: looks good

File: calibration/FLORTM/CGINS-FLORTM-03260__20200317.csv [hunk 3]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+3260,CC_angular_resolution,1.076,
+3260,CC_depolarization_ratio,0.039,
+3260,CC_measurement_wavelength,700,
+3260,CC_scattering_angle,124,
```
Comment (rtravis-whoi) [inline calibration/FLORTM/CGINS-FLORTM-03260__20200317.csv:5]: looks good

File: calibration/DOSTAM/CGINS-DOSTAM-00161__20220309.csv [hunk 4]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+161,CC_calibration_date,1646784000000,sensor calibration date (milliseconds since 1970-01-01)
+161,CC_ref_salinity,0,Optode salinity setting
```
Comment (rtravis-whoi) [inline calibration/DOSTAM/CGINS-DOSTAM-00161__20220309.csv:3]: looks good

File: calibration/CTDGVM/CGINS-CTDGVM-09287__20220401.csv [hunk 5]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9287,CC_calibration_date,1648771200000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/CTDGVM/CGINS-CTDGVM-09287__20220401.csv:2]: looks good

File: calibration/DOSTAM/CGINS-DOSTAM-00414__20210208.csv [hunk 6]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+414,CC_calibration_date,1612742400000,sensor calibration date (milliseconds since 1970-01-01)
+414,CC_ref_salinity,0,Optode salinity setting
```
Comment (rtravis-whoi) [inline calibration/DOSTAM/CGINS-DOSTAM-00414__20210208.csv:3]: looks good

File: calibration/FLORTM/CGINS-FLORTM-03815__20220328.csv [hunk 7]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+3815,CC_angular_resolution,1.076,
+3815,CC_depolarization_ratio,0.039,
+3815,CC_measurement_wavelength,700,
+3815,CC_scattering_angle,124,
```
Comment (rtravis-whoi) [inline calibration/FLORTM/CGINS-FLORTM-03815__20220328.csv:5]: looks good

File: calibration/FLORTO/CGINS-FLORTO-01332__20220329.csv [hunk 8]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+1332,CC_angular_resolution,1.076,
+1332,CC_depolarization_ratio,0.039,
+1332,CC_measurement_wavelength,700,
+1332,CC_scattering_angle,124,
```
Comment (rtravis-whoi) [inline calibration/FLORTO/CGINS-FLORTO-01332__20220329.csv:5]: looks good

File: calibration/PARADM/CGINS-PARADM-50200__20220502.csv [hunk 9]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50200,CC_bsipar_par_scaling,1,
```
Comment (rtravis-whoi) [inline calibration/PARADM/CGINS-PARADM-50200__20220502.csv:2]: looks good

**Review by rtravis-whoi — APPROVED** [review]

File: calibration/CTDGVM/CGINS-CTDGVM-09236__20140923.csv (hunk 1 above)
Comment (snwhite) [reply calibration/CTDGVM/CGINS-CTDGVM-09236__20140923.csv:2]: Looks good.

File: calibration/CTDGVM/CGINS-CTDGVM-09236__20200213.csv (hunk 2 above)
Comment (snwhite) [reply calibration/CTDGVM/CGINS-CTDGVM-09236__20200213.csv:2]: Looks good.

File: calibration/CTDGVM/CGINS-CTDGVM-09287__20220401.csv (hunk 5 above)
Comment (snwhite) [reply calibration/CTDGVM/CGINS-CTDGVM-09287__20220401.csv:2]: Looks good.

File: calibration/DOSTAM/CGINS-DOSTAM-00161__20220309.csv (hunk 4 above)
Comment (snwhite) [reply calibration/DOSTAM/CGINS-DOSTAM-00161__20220309.csv:3]: Looks good.

File: calibration/DOSTAM/CGINS-DOSTAM-00414__20210208.csv (hunk 6 above)
Comment (snwhite) [reply calibration/DOSTAM/CGINS-DOSTAM-00414__20210208.csv:3]: Looks good.

File: calibration/FLORTO/CGINS-FLORTO-01332__20220329.csv (hunk 8 above)
Comment (snwhite) [reply calibration/FLORTO/CGINS-FLORTO-01332__20220329.csv:5]: Looks good.

File: calibration/FLORTM/CGINS-FLORTM-03815__20220328.csv (hunk 7 above)
Comment (snwhite) [reply calibration/FLORTM/CGINS-FLORTM-03815__20220328.csv:5]: Looks good.

File: calibration/FLORTM/CGINS-FLORTM-03260__20200317.csv (hunk 3 above)
Comment (snwhite) [reply calibration/FLORTM/CGINS-FLORTM-03260__20200317.csv:5]: Looks good.

**Review by snwhite — APPROVED** [review]
All look good.

**Pull request merged**